lib/bemxjst/tree.js

```diff
@@ -183,6 +178,7 @@ AddMatch.prototype.mixWrapBody = function mixWrapBody(body) {
   if (typeof body !== 'function') {
     return function inlineAddMixAdaptor() {
       var ret = apply('mix');
+      /* istanbul ignore else */
       if (!Array.isArray(ret)) ret = [ ret ];
```

Comment:
How? See 187 line. We need to concatenate ret and body.

You mean always do `[ ret ].concat(body)`?